### ISSUE TITLE
invalid stepname does not log error for prometheus forwarder

### ISSUE THREAD

**Issue by cliffle** [sakuli/sakuli-enterprise-forwarder#68]
When I execute the following test case with invalid characters (for Testcase and endOfStep) for the Prometheus forwarder nothing is logged. This was tested with Sakuli `2.3.1-next.278`/prometheus-forwarder `2.3.1-next.109` and the latest docker container.

```javascript
(async () => {
  const testCase = new TestCase("Testcase Ä");
  try {
    await _navigateTo("https://sakuli.io");
    await testCase.endOfStep("Startpage Ä")
  } catch (e) {
    await testCase.handleException(e);
  } finally {
    await testCase.saveResult();
  }
})();
```

The output in `sakuli.log`
```
[Wed Aug 19 2020 10:11:12 GMT+0200 (Central European Summer Time)] INFO: Loaded Sakuli with prometheusPreset,legacyPreset
[Wed Aug 19 2020 10:11:12 GMT+0200 (Central European Summer Time)] DEBUG: Prometheus Forwarder Configuration:
enabled: true,
apiHost: localhost,
apiPort: 9091,
apiJob: pushgateway
[Wed Aug 19 2020 10:11:12 GMT+0200 (Central European Summer Time)] INFO: Started Execution
[Wed Aug 19 2020 10:11:13 GMT+0200 (Central European Summer Time)] INFO: Started Testsuite integrationtest
[Wed Aug 19 2020 10:11:13 GMT+0200 (Central European Summer Time)] INFO: Started Testcase Testcase Ä
[Wed Aug 19 2020 10:11:13 GMT+0200 (Central European Summer Time)] INFO: Started Step UNNAMED
[Wed Aug 19 2020 10:11:13 GMT+0200 (Central European Summer Time)] TRACE: Start action navigateTo
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] TRACE: Finish action navigateTo successfully after 2.048s
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] INFO: Finished Step Startpage Ä with state Ok 2.05s
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] INFO: Started Step UNNAMED
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] INFO: Finished Step UNNAMED with state Ok 0.001s
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] INFO: Finished Testcase Testcase Ä with state Ok 2.053s
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] INFO: Finished Testsuite integrationtest with state Ok 2.122s
[Wed Aug 19 2020 10:11:15 GMT+0200 (Central European Summer Time)] DEBUG: Closed webdriver
```

**Comment by svettwer** [comment]
Root cause analysis required

**Comment by svettwer** [comment]
Seems to be related to https://github.com/sakuli/sakuli-enterprise-forwarder/issues/47

**Comment by svettwer** [comment]
Please try
```typescript
(async () => {
  const testCase = new TestCase("Testcase.");
  try {
    await _navigateTo("https://sakuli.io");
    await testCase.endOfStep("Startpage Ä")
  } catch (e) {
    await testCase.handleException(e);
  } finally {
    await testCase.saveResult();
  }
})();
```
as I assume that umlauts might cause the issue if they are present in the testcase.

**Comment by svettwer** [comment]
okay, same result

**Comment by svettwer** [comment]
Not related to prometheus forwarder. Even without any forwarder configured, `Closed webdriver` is the last log message transported.

**Comment by cliffle** [comment]
closed in favor of sakuli/sakuli#470
